app/src/main/java/com/android/sample/model/map/PermissionChecker.kt

```diff
@@ -0,0 +1,5 @@
+package com.android.sample.model.map
```

Comment:
LGTM!

Reply:
I am not sure this new class u=is useful but there is no problem